docs/Introduction/getting-started/conceptual-overview.md

```diff
@@ -0,0 +1,176 @@
++ [What is Remix?](#what-is-remix)
++ [What is Metamask?](#what-is-metamask)
+
+## What is a smart contract? What is a hybrid smart contract?
+
+When deployed to a blockchain, a *smart contract* is a set of instructions that can be executed without intervention from third parties. The code of a smart contract determines how it responds to input, just like the code of any other computer program.
+
+A valuable feature of smart contracts is that they can store and manage on-chain assets (like [ETH or ERC20 tokens](https://ethereum.org/en/developers/docs/standards/tokens/erc-20/)), just like you can with an Ethereum wallet. Because they have an on-chain address like a wallet, they can do everything any other address can. This opens the door for programming automated actions when receiving and transferring assets.
+
+Smart contracts can connect to real-world market prices of assets to produce powerful applications. Being able to securely connect smart contracts with off-chain data and services is what makes them *hybrid* smart contracts. This is done using oracles.
+
+## What language is a smart contract written in?
+
+The most popular language for writing smart contracts on Ethereum is [Solidity](https://docs.soliditylang.org/en/v0.8.7/). It was created by the Ethereum Foundation specifically for smart contract development and is constantly being updated.
+
+If you've ever written Javascript, Java, or other object-oriented scripting languages, Solidity should be easy to understand. Similar to object-oriented languages, Solidity is considered to be a *contract*-oriented language.
+
+## What does a smart contract look like?
+
+The structure of a smart contract is similar to that of a class in Javascript, with a few differences. For example, the following `HelloWorld` contract is a simple smart contract that stores a single variable and includes a function to update the value of that variable.
+
+```solidity
+{% include samples/Tutorials/HelloWorld.sol %}
+```
+
+<div class="remix-callout">
+  <a href="https://remix.ethereum.org/#url=https://docs.chain.link/samples/Tutorials/HelloWorld.sol" target="_blank" >Deploy using Remix</a>
+  <a href="/docs/conceptual-overview/#what-is-remix" >What is Remix?</a>
+</div>
+
+### Solidity versions
+
+The first thing that every Solidity file must have is the Solidity version definition. The `HelloWorld.sol` contract uses version `0.8.7`, which is defined in the contract as `pragma solidity 0.8.7;`
+
+You can see the latest versions of the Solidity compiler [here](https://github.com/ethereum/solc-bin/blob/gh-pages/bin/list.txt/?target=_blank). You might also notice smart contracts that are compatible with a range of versions.
+
+```solidity
+pragma solidity >=0.7.0 <0.9.0;
+```
+This means that the code is written for Solidity version 0.7.0, or a newer version of the language up to, but not including version 0.9.0. In short, `pragma` is used to instruct the compiler as how to treat the code.
```

Comment:
This isn't quite right, it's actually used to CHOOSE the compiler.

Reply:
Updated this sentence to be more accurate.